### PR TITLE
update parent pom to 4.53

### PR DESCRIPTION
**Pull request by Vandit1604**
allow compile java 11

- [X] Make sure you are opening from a **topic/feature/bugfix branch** (right side) and not your main branch!
- [X] Ensure that the pull request title represents the desired changelog entry
- [X] Please describe what you did

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -25,11 +25,12 @@
     </licenses>
 
     <properties>
-        <revision>1.0</revision>
-        <changelist>-SNAPSHOT</changelist>
-        <jenkins.version>2.277.1</jenkins.version>
-        <java.level>8</java.level>
-        <gitHubRepo>jenkinsci/${project.artifactId}-plugin</gitHubRepo>
+        <spotbugs.effort>Max</spotbugs.effort>
+		<spotbugs.threshold>Low</spotbugs.threshold>
```
Comment (Abingcbc) [inline pom.xml:29]: Redundant indent

Comment (Abingcbc) [inline pom.xml:29]: Redundant indent

File: pom.xml [hunk 2]
```diff
@@ -5,7 +5,7 @@
     <parent>
         <groupId>org.jenkins-ci.plugins</groupId>
         <artifactId>plugin</artifactId>
-        <version>4.32</version>
+        <version>4.53</version>
```
Comment (Abingcbc) [inline pom.xml:8]: The dependabot tried to bump the version, but the CI has failed https://github.com/jenkinsci/casdoor-auth-plugin/pull/5.
Have you tested this plugin can be built with 4.53?

Comment (Vandit1604) [reply pom.xml:8]: >Have you tested this plugin can be built with 4.53?
>
Yes i did, i ran `mvn clean verify` and `mvn clean test` and both builds succeeded

File: pom.xml (hunk 1 above)
Comment (Vandit1604) [reply pom.xml:29]: >Redundant indent
>
I'll fix it

Comment (Vandit1604) [reply pom.xml:29]: >Redundant indent
>
Done

Comment (Abingcbc) [reply pom.xml:29]: Sorry, it's my mistake. The indent is necessary.

Comment (Abingcbc) [reply pom.xml:29]: There is a conflict on these codes. Could you resolve conflicts? Then, we can merge the PR :)

Comment (Vandit1604) [reply pom.xml:29]: >Sorry, it's my mistake. The indent is necessary.
>
I formatted the code with prettier so i think its alright now 
feel free to suggest change if it's still wrong

Comment (Vandit1604) [reply pom.xml:29]: >There is a conflict on these codes. Could you resolve conflicts? Then, we can merge the PR :)
>
Working on it right now

**Pull request merged**